### PR TITLE
Support for OpenGL ES and Vulkan tools

### PR DESCRIPTION
**Pull request by ahsanhussain**

### CODE REVIEW

**Comment by Noor-Ahsan** [comment]
kindly add headers to the files

File: meta-flex-distro/conf/distro/flex-os.conf [hunk 1]
```diff
@@ -214,8 +214,9 @@ IMAGE_ROOTFS_EXTRA_SPACE:append = "${@bb.utils.contains('DISTRO_FEATURES', 'virt
 FEATURE_PACKAGES_flex-weston = "packagegroup-core-weston"
 FEATURE_PACKAGES_x11-base ?= "packagegroup-core-x11-base"
 FEATURE_PACKAGES_flex-x11 = "${FEATURE_PACKAGES_x11-base}"
-FEATURE_PACKAGES_graphics += "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '${FEATURE_PACKAGES_flex-weston}', '', d)}"
+FEATURE_PACKAGES_graphics += "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '${FEATURE_PACKAGES_flex-weston} waylandeglinfo', '', d)}"
 FEATURE_PACKAGES_graphics += "${@bb.utils.contains('DISTRO_FEATURES', 'x11', '${FEATURE_PACKAGES_flex-x11}', '', d)}"
+FEATURE_PACKAGES_graphics += "${@bb.utils.contains('DISTRO_FEATURES', 'vulkan', 'vulkan-loader vulkan-tools glslang spirv-tools vulkan-validation-layers', '', d)}"
```
Comment (hashraf91) [inline meta-flex-distro/conf/distro/flex-os.conf:219]: Are all of these packages for `vulkan`? The `glslang` seems related to `opengl`. We should condition that with `opengl` distro feature.

File: meta-flex-distro/recipes-graphics/assimp/assimp_5.3.1.bb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    -D ASSIMP_DOUBLE_PRECISION=OFF \
+    -D ASSIMP_OPT_BUILD_PACKAGES=OFF \
+    -D ASSIMP_BUILD_ASSIMP_TOOLS=OFF \
+    -D ASSIMP_BUILD_SAMPLES=OFF \
+    -D ASSIMP_BUILD_ZLIB=OFF \
+    -D ASSIMP_BUILD_TESTS=OFF \
+    -D ASSIMP_WARNINGS_AS_ERRORS=ON \
+    -D ASSIMP_IGNORE_GIT_HASH=OFF \
+    -D ASSIMP_LIB_INSTALL_DIR=${baselib} \
+"
```
Comment (hashraf91) [inline meta-flex-distro/recipes-graphics/assimp/assimp_5.3.1.bb:39]: assimp is giving this warning:
```
WARNING: assimp-5.3.1-r0 do_package_qa: QA Issue: File /usr/lib/cmake/assimp-5.3/assimpTargets.cmake in package assimp-dev contains reference to TMPDIR [buildpaths]
```

File: meta-flex-distro/conf/distro/flex-os.conf (hunk 1 above)
Comment (ahsanhussain) [reply meta-flex-distro/conf/distro/flex-os.conf:219]: This is for using glsl shaders with Vulkan which is quite probable if we are to run any Vulkan demos

**Review by hashraf91 — APPROVED** [review]

File: meta-flex-distro/recipes-graphics/assimp/assimp_5.3.1.bb (hunk 2 above)
Comment (ahsanhussain) [reply meta-flex-distro/recipes-graphics/assimp/assimp_5.3.1.bb:39]: Fixed

**Pull request merged**